behave/features/steps/assign_case.py

```diff
@@ -100,7 +100,7 @@ def step_impl(context):
 
     def wait_until_diversity_is_complete(*args):
         try:
-            return "The client has completed diversity monitoring." in page.find_by_class("SummaryBlock-content").text
+            return "The client has completed diversity monitoring." in page.find_by_class("SummaryBlock").text
```

Comment:
can you add the reason for this change into the commit message/code as not clear it is linked to this ticket.

Reply:
Added the following to the ticket description:
| Use SummaryBlock css selector for matching diversity questions because the SummaryBlock-content element is being removed and re-added as we progress through each diversity question which can make it a bit tricky to find sometimes